packages/backend/src/core/discovery/utils/config.test.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'earljs'
+
+import { ConfigReader } from '../ConfigReader'
+
+export {}
```

Comment:
is this needed?

Reply:
no, in the beginning file was in different folder and needed this one, but I decided to move it, and forgot to remove this statement, thanks